shell/components/ExplorerMembers.vue

```diff
@@ -204,6 +216,47 @@ export default {
         }
 
         if (roleTemplate) {
+          // we're already looping through each roleTemplate, might as well isolate the ones that belong to the current user and make them easy to find against the project
+          if (isCurrentUser) {
+            const { rules } = roleTemplate;
```

Comment:
this all seems over the top given we're only checking for verbs in a single group, it takes a while to understand what's happened and to debug when something goes wrong.

it also has issues with scaling. if there are many roles with many resources it will impact the parent for loop (which we know can be problematic. if we're only looking for that single group with verbs it we can exit early, and only store a bool for the entire project